### PR TITLE
Update photo endpoints

### PR DESCRIPTION
**Pull request by TimEngleSF**

### CODE REVIEW

File: backend/jsconfig.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "compilerOptions": {
-    "module": "ESNext",
-    "target": "ESNext"
-  },
-  "include": ["**/*.js", "**/*.mjs"],
-  "exclude": ["node_modules"]
+    "compilerOptions": {
```
Comment (JMStudiosJoe) [inline backend/jsconfig.json:2]: did these need a spacing fix?

File: backend/models/BlacklistToken.js [hunk 2]
```diff
@@ -10,52 +10,52 @@ const __filename = fileURLToPath(import.meta.url);
...
-      });
-    } catch (error) {
-      throw await errorHelpers.transformDatabaseError(
-        error,
-        __filename,
-        'addTokenToList',
-      );
-    }
-  },
+const blackListToken = {
```
Comment (JMStudiosJoe) [inline backend/models/BlacklistToken.js:13]: first lets fix spacing, then lets change the naming from blackList to blockList and others blacklistCollection to blockListCollection, please.

File: backend/models/BlacklistToken.js [hunk 3]
```diff
@@ -10,52 +10,52 @@ const __filename = fileURLToPath(import.meta.url);
...
-  },
+    getToken: async (token) => {
+        try {
+            return await blacklistCollection.findOne(
+                { token },
+                { projection: { token: 1 } },
+            );
+        } catch (error) {
+            throw await errorHelpers.transformDatabaseError(
+                error,
```
Comment (JMStudiosJoe) [inline backend/models/BlacklistToken.js:37]: spacing fix here too

**Review by JMStudiosJoe — CHANGES_REQUESTED** [review]
i see the spacing is 4 in some places and 2 in others lets keep it consistent but that should be its own pr fix. But the naming yes please change.

File: backend/models/BlacklistToken.js (hunk 3 above)
Comment (TimEngleSF) [reply backend/models/BlacklistToken.js:37]: I made the updates requested, I have actually implemented refresh and access tokens to be used this week. The plan is to remove the block list.

Once I have verification via email setup, I plan to make another PR which will include the new refreshToken functionality including testing for the model. I can make the PR with the included refresh token functionality before I implement the email verification  if you would like.

You  can let me know on discord, I am TimE in the litter.ai room

Thanks!

Comment (JMStudiosJoe) [reply backend/models/BlacklistToken.js]: Wonderful. Ok sounds great. We are getting organized way more on ops side and might have more options there for volunteers to help as well I will keep you updated. Email setup might take some time, I know David has a domain ready to go and we should have more bandwidth to get these streamlined soon.

**Review by JMStudiosJoe — APPROVED** [review]

**Pull request merged**